qemu/tests/hv_vapic_test.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import re
+import os
+import time
+
+from virttest import error_context
+from virttest import utils_disk
+from virttest import env_process
+from virttest import data_dir
+from avocado.utils import process
```

Comment:
`from avocado.utils import process` is avocado package, should sperate from others.
e.g:
```
from avocado.utils import process

from virttest import error_context
from virttest import utils_disk
from virttest import env_process
from virttest import data_dir
from virttest.qemu_storage import QemuImg

from provider.storage_benchmark import generate_instance
```
more clearly